### PR TITLE
feat: es2015 bundle

### LINKED ISSUE #1037: Class constructor OverlayContainer cannot be invoked without 'new'
**I'm submitting a ...**  (check one with "x")

* [X] bug report
* [ ] feature request

### Issue description

**Current behavior:**
When trying to set target to es6 in tsconfig, the application compiles, however this error is reported on startup.

Class constructor OverlayContainer cannot be invoked without 'new'

**Expected behavior:**
No error

**Steps to reproduce:**

"target": "es6",
    "lib": ["es6",
      "es2017",
      "dom"
    ],

```

### Other information:
Chrome and FireFox

### PR DESCRIPTION
**Pull request by yggg**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:
Adds es2015 bundles. Fixes #1037.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +33,11 @@
     "docs:gh-pages": "npm run docs:build && npm run ngh -- --dir ./docs/dist",
     "build": "ng build",
     "build:wp": "npm run build -- playground-wp --prod --aot",
-    "build:package": "npm-run-all -s clean:tmp:lib gulp build:ts build:schematics:package build:inline-resources build:bundle clean:tmp",
-    "build:ts": "tsc -p tsconfig.publish.json && ngc -p tsconfig.publish.json",
+    "build:package": "npm-run-all -s clean:tmp:lib gulp build:ts build:schematics:package build:inline-resources build:move-es2015-bundle build:bundle clean:tmp",
+    "build:ts": "npm-run-all -p build:ts:es5 build:ts:es2015",
+    "build:ts:es5": "ngc -p tsconfig.publish.json",
+    "build:ts:es2015": "ngc -p tsconfig.publish.json -t ES2015 -outDir src/.lib/es2015",
```
Comment (nnixaa) [inline package.json]: should we also add a test run pointing to es6 bundle version?

Comment (yggg) [reply package.json]: Added in [558e27f](/akveo/nebular/pull/1249/commits/558e27f02ced6008975f681f0f7eb89d27eb88bf)

**Comment by gjsmith66** [comment]
Any ETA when this can be merged and released?

**Comment by yggg** [comment]
Fixed in #1474

**Pull request closed without merging**